augment_manta_vcf/1.0/augment_manta_vcf.py

```diff
@@ -0,0 +1,410 @@
+#!/usr/bin/env python3
+
+"""
+augment_manta_vcf.py
+====================
+
+This script augments Manta VCF files with additional information. Namely:
+    1) TR (FORMAT field): Sum of the SR and PR FORMAT fields per allele.
+    2) DP (FORMAT field): Sum of the SR and PR FORMAT fields across all alleles.
+    3) VAF (FORMAT field): Variant allele fraction for the alternate allele.
+    4) REGIONS (INFO field): List of regions from the given BED files that
+       overlap each variant position.
+
+If also updates the sample IDs in the VCF file with any values specified using
+the `--tumour-id` and `--normal-id` arguments based on the "VCF type" with the
+following rules:
```

Comment:
Do you mean that the user should only include a --tumor-id and/or --normal-id based on these rules? If I include both in a file with only one, will it give me an error?

Reply:
What I wrote here isn't super clear. Here's an edited version:
```
The script can optionally update the sample IDs in the VCF header, namely
the row with "#CHROM". Unfortunately, different Manta output VCF files
include different combinations of the tumour and/or normal samples used
in the analysis. Based on existing Manta output files, it was found that
the sample present in the VCF files follow the rules below. The script
will attempt to infer the "VCF type" (e.g., "somaticSV") based on the
input VCF file path. If this isn't possible for whatever reason, the
user can provide a value for the `--vcf_type` argument. Depending on
the VCF type, the user will need to provide values for `--tumour_id`
and/or `--normal_id`.
```